features/IFC102_Absence-of-deprecated-entities.feature

```diff
@@ -5,33 +5,38 @@
 Feature: IFC102 - Absence of deprecated entities
 
 This rule verifies that the IFC model does not have deprecated entities, attributes or enumerators. 
+IFC2X3 : https://standards.buildingsmart.org/IFC/RELEASE/IFC2x3/TC1/HTML/deprecated_constructs.htm
+IFC4X3: https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/content/introduction.htm
 
-Background: 
-  Given A model with Schema "IFC4.3"
+  Scenario Outline: Check for deprecated entities - IFC4.3
 
-  Scenario Outline: Check for deprecated entities
-  
+    Given A model with Schema "IFC4.3"
     Given An IFC model
 
     Then There must be less than 1 instance(s) of <entity>
 
     Examples:
       | entity | 
```

Comment:
```suggestion
      | entity | 
      | IfcBeamStandardCase |
```

Reply:
Just realized - maybe we don't need this in a gherkin rule since it would be caught first by the schema check.